patches/no-madvise-prototype.patch

```diff
@@ -0,0 +1,21 @@
+Remove madvise prototype to avoid conflicting types.
+
+diff --git a/deps/v8/src/base/platform/platform-posix.cc b/deps/v8/src/base/platform/platform-posix.cc
+index 664ed301c8..cb2bf6958a 100644
+--- a/deps/v8/src/base/platform/platform-posix.cc
++++ b/deps/v8/src/base/platform/platform-posix.cc
+@@ -72,14 +72,6 @@
+ #define MAP_ANONYMOUS MAP_ANON
+ #endif
+ 
+-#if defined(V8_OS_SOLARIS)
```

Comment:
Will this work with the minimum PI set to 20210826T002459Z ?  The mman.h cleanup didn't happen until illumos#14418, which landed in -gate in 2022 and in our 20220401 release.

Reply:
Yeh, this works on both sides of the change. We've had this patch in pkgsrc for a while.

Reply:
Cool, please mention this as part of your testing notes in the jira ticket.